### PR TITLE
Elaborate `getting started` section of README

### PR DESCRIPTION
**Pull request by ixxie**
Distinguish  installation, (minimal) configuration and asset handling steps. Add details for getting the plugin working in SvelteKit.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,16 +30,64 @@ Vite:
...
+
+```svelte
+<script>
+  import sceneUrl from '$lib/assets/scene.glb';
+</script>
+```
+
+### 4. Execution on Build
+
+The plugin will optimize assets _at build time_. This means you will not observe results when running `npm run dev`. To test the optimization, run `npm run build` and `npm run preview`.
```
Comment (donmccurdy) [inline README.md:72]: Nit: the current wording is specific to Vite, npm clients, and particular choices of macro names. Maybe we can clarify it as —

```suggestion
The plugin will optimize assets _at build time_. Vite has [several modes](https://vitejs.dev/guide/env-and-mode.html#modes), and optimization occurs in production (`vite build`) and preview (`vite preview`) but not development (`vite`) modes.
```

**Review by donmccurdy — COMMENTED** [review]
a couple suggestions, but this looks great – thank you!

**Comment by ixxie** [comment]
@donmccurdy thanks for the suggestions... of course my knowledge is limited here so its nice you filled in the gaps.

I squashed everything together and added you as a coauthor in the commit message.

**Review by juniorxsound — APPROVED** [review]
LGTM - thanks @ixxie! @donmccurdy thoughts?

File: README.md [hunk 2]
```diff
@@ -28,16 +30,68 @@ Vite:
...
+    gltf({
+      transforms: [draco()],
+    }),
+  ],
+};
+```
+
+See [advanced configuration](#advanced-configuration) for more complex examples.
+
+_SvelteKit:_ it seems that in SvelteKit SSR should be disabled in order to avoid [mysterious `ReferenceError` issues](https://github.com/nytimes/rd-bundler-3d-plugins/issues/14#issuecomment-1410620469).
```
Comment (donmccurdy) [inline README.md:56]: Would you mind filing an issue (on this repo) about SvelteKit SSR too? Does seem very weird that our plugin would affect that. 🫤

**Review by donmccurdy — APPROVED** [review]
One optional suggestion / looks good to me!

File: README.md (hunk 2 above)
Comment (ixxie) [reply README.md]: I created the issues and referenced them in the README:

https://github.com/sveltejs/kit/issues/9000
https://github.com/nytimes/rd-bundler-3d-plugins/issues/19

**Pull request merged**